.azure_pipelines/olive-examples.yaml

```diff
@@ -0,0 +1,67 @@
+trigger: none
+
+pr: none
+
+schedules:
+- cron: '0 0 * * 1'
```

Comment:
do you think weekly will be enough? Since we are actively developing the code, the examples might break frequently.

Reply:
also we might need another weekly run to test the examples with the nightly ort package also. unless we pin the ort requirements for the tests

Reply:
I think once a week is enough, even daily we don't check it daily. Once a week we can have a weekly schedule to check the pipeline. As for nightly version, I checked with Devang and he said we don't need it for now. We could add it later when we need it.

Reply:
sounds good. thanks!